src/__tests__/extractMeta.js

```diff
@@ -1,6 +1,9 @@
 import extractMetadata from '../extractMeta';
 import ElementPropTypes from '../propTypes';
 
+import { listAvailableComponents }from "@volusion/element-components";
```

Comment:
```suggestion
import { listAvailableComponents } from "@volusion/element-components";
```
lint not running on tests I take it?